src/Behat/Behat/DependencyInjection/config/behat.xml

```diff
@@ -305,8 +305,10 @@
                 <service class="%behat.translator.message_selector.class%" />
             </argument>
 
-            <call method="setFallbackLocale">
-                <argument>en</argument>
+            <call method="setFallbackLocales">
```

Comment:
OK, bumped to `~2.3`.  Do you think it makes sense to bump the other symfony requirements explicitly as well? `symfony/config` and `symfony/yaml` are implicitly bumped due to https://github.com/symfony/Translation/blob/2.3/composer.json